src/utils/statuses/getCurrentStatuses.ts

```diff
@@ -0,0 +1,28 @@
+import { Status } from "pages/patch/patchTabs/tasks/TaskFilters";
```

Comment:
I dont think this is general enough to  live inside the `utils` folder. I think its only used by the tasks filter component, So it should probably live near it.

Reply:
This function can be used for the task test statuses too

Reply:
Didnt read your comment before posting this review sorry.